pkgs/by-name/he/hextazy/package.nix

```diff
@@ -0,0 +1,35 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  rustPlatform,
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "hextazy";
+  version = "0.2";
+
+  src = fetchFromGitHub {
+    owner = "faelian";
+    repo = "hextazy";
+    rev = "${version}";
+    hash = "sha256-e1uoKMejdTz8KH7uFkZ9YCE1WTwaxCZjcxh4g2ermNY=";
+  };
+
+  cargoHash = "sha256-5Z+ptr1JmeiP4hK8k+M1d189I0X53/jRUEFwgWGK3xQ=";
+
+  doInstallCheck = true;
```

Comment:
What happens in installCheck that you did not have set?

Reply:
I just copied it, I'll remove it and try without, I thought it may need to be explicitly set but I'm most likely wrong.